src/Aspire.Hosting.Azure.ServiceBus/ApplicationModel/ServiceBusClientAffineProperties.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.Azure.ServiceBus.ApplicationModel;
```

Comment:
Do we really need an `ApplicationModel` namespace? Can't these types just go under `Aspire.Hosting.Azure.ServiceBus`?